### ISSUE TITLE
[bug]: Transactional Emails blocked by MS365

### ISSUE THREAD

**Issue by seanvandermolen** [makeplane/plane#4363]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current behavior

Transactional emails during sign-up and invitations are so far blocked by MS365/Outlook.  Requires opening in a browser in plaintext.

![image](https://github.com/makeplane/plane/assets/7504775/30b87390-cb39-4408-86d7-fd796dfd02bd)
![image](https://github.com/makeplane/plane/assets/7504775/b5fe3c45-bfbb-44f3-8265-15fe9ddf1dd4)
![image](https://github.com/makeplane/plane/assets/7504775/8bc7a2bd-5bc1-499a-a20d-2da2540f1e64)

### Steps to reproduce

Any action that triggers a transactional email.
Sign up email confirmation.
Add a member.

### Environment

Production

### Browser

Other

### Variant

Cloud

### Version

v0.19.0

**Comment by srinivaspendem** [comment]
Closing as duplicate of #3184
